ui/app/components/app/modals/hide-token-confirmation-modal.js

```diff
@@ -25,57 +24,64 @@ function mapDispatchToProps (dispatch) {
   }
 }
 
-inherits(HideTokenConfirmationModal, Component)
-function HideTokenConfirmationModal () {
-  Component.call(this)
-
-  this.state = {}
-}
-
-HideTokenConfirmationModal.contextTypes = {
-  t: PropTypes.func,
-}
+class HideTokenConfirmationModal extends Component {
+  static contextTypes = {
+    t: PropTypes.func,
+  }
 
-export default connect(mapStateToProps, mapDispatchToProps)(HideTokenConfirmationModal)
+  static propTypes = {
+    network: PropTypes.number,
```

Comment:
```suggestion
    network: PropTypes.string,
```
This should be a string; it starts with the value 'loading', and the other actual `chainId` numbers are all string as well.

Reply:
Turns Out™ that this prop isn't used by the `Identicon` component